ui/ducks/send/send.js

```diff
@@ -511,6 +514,7 @@ export const initialState = {
     balance: '0x0',
   },
   stage: SEND_STAGES.INACTIVE,
+  swapsBlockedTokens: [],
```

Comment:
Should this be an object?

Reply:
storing the `blockedTokens` responses per chain can add bloat to state and I don't see a reason why we would need access to multiple chain lists at once (yet). the api responses are cached for 5 mins in the browser so the extension won't be hitting the endpoint each time the page loads or the user switches chains anyways

Reply:
Fair enough